### DISCUSSION TITLE
使用ant design的Image组件无法正常显示页面

### DISCUSSION THREAD

**Discussion by XCGB** [ant-design/ant-design#42805] [Q&A]
下面可以正常实现
`  {
    title: '头像',
    dataIndex: 'avatarUrl',
    render: (_, record) => (
      <div>
        <img src={record.avatarUrl} width={100}></img>
      </div>
    ),
  },
  `
换成Image就无法实现，
`  import { Image } from 'antd';
{
    title: '头像',
    dataIndex: 'avatarUrl',
    render: (_, record) => (
      <div>
        <Image src={record.avatarUrl} width={100}/> 
      </div>
    ),
  },
`
 antd 的版本信息    "antd": "^4.20.0",

**Comment by kiner-tang** [comment]
官网示例看起来是可以的，可以 fork 一下这个示例，给一个能够复现问题的最小示例吗？
https://codesandbox.io/s/po21bu?file=/demo.tsx

**Comment by XCGB** [comment]
很抱歉我不知道如何复现最小演示实例，我是通过Ant Design Pro官方文件生成的使用  umi@3的simple 的脚手架，创建了一个这样的表格https://codesandbox.io/s/wiy67d
修改其中的APP.tsx的文件中 columns 的属性使用下面的方式创建了用于查看图片的样式

```
  {
    title: '头像',
    dataIndex: 'avatarUrl',
    render: (_, record) => (
      <div>
        <Image src={record.avatarUrl} width={100}/>
      </div>
    ),
  },
```

**Reply by kiner-tang** [reply]
<img width="1321" alt="image" src="https://github.com/ant-design/ant-design/assets/10286961/6adf8fc4-ed95-4eb4-948a-e980ebad2c88">
接口返回的头像字段是`avatar`，并不是`avatarUrl`。
https://codesandbox.io/s/cha-xun-biao-ge-forked-k2bkoe

**Reply by XCGB** [reply]
这里定义的字段是我在API中声明的字段，所以修改成了avatarUrl，似乎并不影响运行，因为使用`<img>`可以正常读取。
![image](https://github.com/ant-design/ant-design/assets/127387227/e52286ce-55fa-414d-aac8-cd4966bcbc2b)
我在思考是否是因为版本问题呢。
![image](https://github.com/ant-design/ant-design/assets/127387227/790c8108-5b1f-45e4-a4d6-c7d7e0b0cdd6)

**Reply by kiner-tang** [reply]
应该跟版本关系不大，我看 4.20.7 是可以正常展示的，你看看会不会你项目中有其他代码影响到了。最好能提供一个最小的在线复现示例，方便排查问题

**Reply by XCGB** [reply]
这是文件中的关于package.json参数版本，
https://codesandbox.io/s/cha-xun-biao-ge-forked-qke1n0?file=/package.json
